test/model.user-identity.test.js

```diff
@@ -244,4 +244,57 @@ describe('UserIdentity', function() {
       done();
     });
   });
+
+  describe('emailOptional set to false', function(err) {
+    function customProfileToUser(provider, profile, options) {
+      var userInfo = {
+        username: profile.username,
+        password: 'secret',
+      };
+      if (profile.emails && profile.emails[0] && profile.emails[0].value) {
+        userInfo.email = profile.emails[0].value;
+      }
+      return userInfo;
+    }
```

Comment:
Move to bottom of function of describe block please: https://github.com/strongloop/loopback-contributor-docs/blob/master/style-guide.md#good-8

cc @jannyHou